### PR TITLE
Apache: enable AllowEncodedSlashes for VisualEditor

### PR DESCRIPTION
**Pull request by slokhorst**
See https://phabricator.wikimedia.org/T262392 and https://www.mediawiki.org/wiki/Topic:Vevdw6kw6itlr0x7

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ function mediawiki_version() {
 }
 
 declare -A variantExtras=(
-	[apache]='\n# Enable Short URLs\nRUN set -eux; \\\n\ta2enmod rewrite; \\\n\t{ \\\n\t\techo \"<Directory /var/www/html>\"; \\\n\t\techo \"  RewriteEngine On\"; \\\n\t\techo \"  RewriteCond %{REQUEST_FILENAME} !-f\"; \\\n\t\techo \"  RewriteCond %{REQUEST_FILENAME} !-d\"; \\\n\t\techo \"  RewriteRule ^ %{DOCUMENT_ROOT}/index.php [L]\"; \\\n\t\techo \"</Directory>\"; \\\n\t} > \"$APACHE_CONFDIR/conf-available/short-url.conf\"; \\\n\ta2enconf short-url'
+	[apache]='\n# Enable Short URLs\nRUN set -eux; \\\n\ta2enmod rewrite; \\\n\t{ \\\n\t\techo \"<Directory /var/www/html>\"; \\\n\t\techo \"  RewriteEngine On\"; \\\n\t\techo \"  RewriteCond %{REQUEST_FILENAME} !-f\"; \\\n\t\techo \"  RewriteCond %{REQUEST_FILENAME} !-d\"; \\\n\t\techo \"  RewriteRule ^ %{DOCUMENT_ROOT}/index.php [L]\"; \\\n\t\techo \"</Directory>\"; \\\n\t} > \"$APACHE_CONFDIR/conf-available/short-url.conf\"; \\\n\ta2enconf short-url\n\n# Enable AllowEncodedSlashes for VisualEditor\nRUN sed -i \"s/<\\/VirtualHost>/\\tAllowEncodedSlashes NoDecode\\n<\\/VirtualHost>/\" \"$APACHE_CONFDIR/sites-available/000-default.conf\"'
```
Comment (davidbarratt) [inline update.sh:31]: suggest creating a new config file (like the line above) and enabling it rather than modifying the existing default config file.

**Review by davidbarratt — CHANGES_REQUESTED** [review]

File: update.sh (hunk 1 above)
Comment (slokhorst) [reply update.sh]: Thanks, you're right. I wasn't sure if it would work outside of the `<Directory>` block, but it does! Fixed.

Comment (slokhorst) [reply update.sh:31]: I tried this, but it doesn't seem to work outside of the VirtualHost block. Apparently it cannot be set globally, see e.g. https://bz.apache.org/bugzilla/show_bug.cgi?id=46830

**Comment by slokhorst** [comment]
Anything blocking this?

**Pull request merged**